### DISCUSSION TITLE
I would like to be able to have the option to visualise all data in the dB as a button on the database element

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#4349] [Feedback]
A user says:
>It has been fun playing with dark. It took a while to understand the dark way of programming but I’m getting there. One thing I raised in the slack channel was the fact that it would be good to have examples of the method usage within the docs. I was overwhelmed by the number of database methods. If I was working with a traditional database, I would deal with select * or select  x,z and some filter but as I look at the number of dark database methods I am lost as to why I would use the say Getmany over getAll.
>
>I have some issues with the workspace. I work on different screens. I have only got a handful of functions and one database setup. On my larger desktop screen I can float between elements fine. However on my smaller laptop screen it gets harder for me to find the element I am looking for. It is currently a minor issue but I can easily see how it could become a problem if I was to write a more complex api. 
>
>I would like to be able to have the option to visualise all data in the dB as a button on the database element. Why do I have to write a repl function to do that? 
Finally I am starting a slightly more complex piece in dark. I would like the ability to copy data or import data from mongo or at the very least be able to import Json files.
>
>Please do not take my comments as discouragement. I love dark and wouldn’t be on board if I didn’t believe that this was a good solution.

**Comment by StachuDotNet** [comment]
> I would like to be able to have the option to visualise all data in the dB as a button on the database element.

Extending this, I'd like to be able to edit Datastore data within the Editor.
(Especially useful when seeding a bunch of data manually.)
